pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
I'm a bit confused about @Mic92 suggestion to use just the `unstable-20170411` version, without any reference to the current stable version. What are the reasoning behind this?

I think that `7.1-unstable-`... not upgrading to 7.1, but upgrading to 7.2 (whenever it'll be released) is the desired outcome. So the `7.1-unstable-20170411` should be OK, isn't it?